docs/index.rst

```diff
@@ -41,11 +41,13 @@ or are you looking to use your existing docs with Read the Docs?
 Learn about documentation authoring tools such as Sphinx and MkDocs
 to help you create fantastic documentation for your project.
 
+* **Tutorial**: :doc:`/tutorial/index`
```

Comment:
Should we link this from the top and/or bottom of the getting start w/ Sphinx page as well?

Reply:
Good idea! Will do